src/platform/site-wide/sass/modules/_m-medallia.scss

```diff
@@ -97,10 +80,37 @@
 }
 
 /* https://github.com/department-of-veterans-affairs/va.gov-team/issues/18706#issuecomment-769341669 */
+/* style the focus-circles on radio buttons for the survey page */
 .neb-component
   .neb-content
   .neb-native-radio
   input[type="radio"].wcag-radio-border:focus {
   border: 2px solid rgb(249, 198, 66) !important;
   padding: 1px !important;
 }
+
```

Comment:
The following changes were provided to me via email. 

![image](https://user-images.githubusercontent.com/6130520/108786437-3522f080-7539-11eb-8ec9-7af57bdcfbcb.png)